common/scripts/build-linux-sbsa.sh

```diff
@@ -41,7 +41,7 @@ build_sbsa_kernel_driver()
     echo $arch
     if [[ $arch = "aarch64" ]]
```

Comment:
@PhatFree:  Can we have line 42 changed to below as done in build-grub.sh
  if [ $arch == "aarch64" ]; then